### PR TITLE
EXP-5463 - improving bundle

### PR DESCRIPTION
**Pull request by oleg-andreyev**
- moved everything under src/
- created tests/
- added travis

### CODE REVIEW

**Comment by oleg-andreyev** [comment]
@chernecov @inferont @githoober

File: README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ Generic consumer and standard event message model meanth to pull in a domain eve
 Documentation
 -------------
 
-Documentation for Ecentria Rest Bundle is in [Resources/doc/index.md](Resources/doc/index.md)
+Documentation for Ecentria Rest Bundle is in [Resources/doc/index.md](src/Resources/doc/index.md)
```
Comment (githoober) [inline README.md:9]: It's not Ecentria Rest Bundle.

**Comment by githoober** [comment]
Judging based on the types of changes here, this should target the next major version 2.0, correct?

**Comment by oleg-andreyev** [comment]
Yes, it should be 2.x

**Review by githoober — APPROVED** [review]

**Comment by oleg-andreyev** [comment]
@githoober nice one! 💪🥹

**Pull request merged**